GNUmakefile

```diff
@@ -52,13 +53,19 @@ checklicense: $(ADDLICENCESE)
 govulncheck: $(GOVULNCHECK)
 	$(GOVULNCHECK)
 
-build: fmtcheck
-	go install
+.PHONY: lint
+lint: $(GOLANGCI_LINT)
+	$(GOLANGCI_LINT) run -v
 
-test: fmtcheck
-	go test -i $(TEST) || exit 1
-	echo $(TEST) | \
-		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
+.PHONY: lint-fix
+lint-fix:
+	$(GOLANGCI_LINT) run -v --fix
+
+build:
+	go build
```

Comment:
I suspect that this will impact someone's workflow however, I am curious if it is used.